R/scenario-configuration.R

```diff
@@ -111,6 +111,38 @@ a parameter sheet from the list"
       } else {
         stop(messages$errorPropertyReadOnly("projectConfiguration"))
       }
+    },
+    #' @field customFunction A function that will be applied at the very last
+    #' step of simulation initialization. Can be applied for advanced scenario
+    #' configuration. If the function requires additional arguments, they must be
+    #' defined in `customFunctionArgs` as a list. The object `simulation` can always
+    #' be accessed.
+    #' @example \dontrun{
+    #' #This example gets the current value of the "Age" parameter and multiplies
+    #' #it by a factor coming from argument.
+    #' #Assume that scenarioConfiguration has been created from excel.
+    #' scenarioConfiguration$customFunction <- function(
+    #' }
+    customFunction = function(value) {
```

Comment:
Method to add a custom function that will be applied to the simulation object at the end of initialization